QUARKUS-6258.md

```diff
@@ -15,9 +15,16 @@ The list of prioritized features (taken from the issue):
 - Advanced RAG
 - Guardrails 	
 - WebSocket chat integration 	
-- CDI Chat Model.
+- CDI Chat Model
 
-We also need to verify correct usage of OpenAI and Watson.x (optional for the first release) APIs as well as observability (metrics, logging, and telemetry).
+We also need to verify correct usage of OpenAI and Watson.x (optional for the first release) APIs as well as observability (metrics, logging, and telemetry), the latter being a requirement for full support.
+
+### Optional test cases
+These are required only for test-preview status in case Quarkus will go the way of maximum support in te first release:
+- Scoring Model
+- Parsing of documents (PDF, Word, etc)
+- MCP client and server (STDIO and Streamable HTTP )
+- Testing AI infused application (required for dev preview)
```

Comment:
what is the dev preview? I know tech-preview, I know dev support https://access.redhat.com/support/offerings/developer/. I think I have heard it, but I have problem to find it, is this the right term?

Reply:
Not 100% sure but it should be this https://access.redhat.com/support/offerings/devpreview as one of the support level defined in https://access.redhat.com/articles/7040491

Reply:
I got answer in the meeting, thanks